src/network/networkd-routing-policy-rule.c

```diff
@@ -1453,6 +1454,13 @@ int routing_policy_load_rules(const char *state_file, Set **rules) {
                                         log_error_errno(r, "Failed to parse RPDB rule suppress_prefixlen, ignoring: %s", b);
                                         continue;
                                 }
+                        } else if (streq(a, "invert_rule")) {
+                                r = parse_boolean(b);
+                                if (r < 0) {
+                                        log_error_errno(r, "Failed to parse RPDB rule invert_rule, ignoring: %s", b);
```

Comment:
should be downgraded to warning i guess (and the other cases like this too). 

(i.e. if a log message says "ignoring" it's usually a good indication it should be warning, and not error)